pom.xml

```diff
@@ -1,10 +1,8 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <parent>
```

Comment:
@aheritier I adopted the pom structure of sonar-reference-plugin, where parent is missing:
https://github.com/SonarSource/sonar-examples/blob/master/plugins/sonar-reference-plugin/pom.xml

But after checking it's content, I would add it again to keep the additional information. Are you OK with that?

Best Regards